src/pages/home/home.scss

```diff
@@ -0,0 +1,73 @@
+.homeContainer {
+  height: 100%;
+
+  & .mainHeaderBackground {
+    background: url('../../assets/main-header-background.png') no-repeat;
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    position: absolute;
+    z-index: -1;
+  }
+
+  & .mainHeaderContent {
+    display: flex;
+    margin-left: 4em;
+
+    & div {
+      flex: 1;
+
+      & .header {
+        font-family: 'Fjalla One', sans-serif;
+        font-size: 4.25rem;
+        color: #ffffff;
+  
+        &:nth-child(1) {
+          margin-bottom: 0px;
+        }
+  
+        &:nth-child(2) {
+          margin-top: 0px;
+        }
+      }
+
+      & h4 {
+        font-family: 'Lato', sans-serif;
+        font-size: 1.5rem;
+        font-weight: 300;
+        color: #ffffff;
+      }
+
+      & .learnMore {
+        border-radius: 3px;
+        padding: 10px 30px;
+        font-family: 'Lato', sans-serif;
+        font-size: 1rem;
+        border: 0px;
+        text-transform: uppercase;
+        color: #ffffff;
+        background-color: #32B88C;
+      }
+    }
+  }
+}
+
+
+.navigationBar {
+  padding: 1.25em 0em;
+
+  & .leftNavLinks {
```

Comment:
is the `&` required when nesting? I am not sure myself

Reply:
It's required only if you want that class at the same level of the parent, in this case the output will be:

```
.navigationBar.leftNavLinks {

}
```

Without the `&`  it will be

```
.navigationBar     .leftNavLinks {

}
```

Reply:
Okay, that's what I was wondering. I am accustom to `&.className` so wanted to make sure it was the same idea. Thanks @crysfel!

Reply:
Since there is a space between `&` and `.leftNavLinks` it will actually be `.navigationBar .leftNavLinks` as opposed to `.navigationBar.leftNavLinks` 

Anyways, I have made the necessary modifications.